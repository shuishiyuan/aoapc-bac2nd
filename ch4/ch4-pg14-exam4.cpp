#include <stdio.h>

int main() {
    printf("Message Decoding, ACM/ICPC World Finals 1991, UVa 213!");
    return 0;
}
