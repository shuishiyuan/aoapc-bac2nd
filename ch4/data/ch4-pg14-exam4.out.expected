##*\$
0(A),00(B),01(#),10(T),000(A),001(N),010(C),011(n),100(r),101(t),110(X),0000(c),0001, ... ,1101,1110,00000, ... 
